scripts/init-node.sh

```diff
@@ -102,7 +100,7 @@ function run_node() {
     echo "Launching node with: $node_cmd"
     nohup sh -c "$node_cmd" &
     sleep 5
-    cp -H ~/.safe/prefix_maps/default ~/prefix-map
+    cp ~/node_data/prefix_map ~/genesis-prefix-map
```

Comment:
If you change the name of `~/prefix-map` to `~/genesis-prefix-map` you need to also make changes to `genesis.tf`. I would probably just leave the destination file name as it was.